src/clr-angular/utils/a11y/aria-live.service.ts

```diff
@@ -0,0 +1,95 @@
+/*
+ * Copyright (c) 2016-2019 VMware, Inc. All Rights Reserved.
+ * This software is released under MIT license.
+ * The full license information can be found in LICENSE in the root directory of this project.
+ */
+
+import { DOCUMENT } from '@angular/common';
+import { Inject, Injectable, NgZone, OnDestroy } from '@angular/core';
+import { uniqueIdFactory } from '../id-generator/id-generator.service';
+
+// export type AriaLivePoliteness = 'off' | 'polite' | 'assertive';
+export enum AriaLivePoliteness {
+  off = 'off',
+  polite = 'polite',
+  assertive = 'assertive',
+}
+
+@Injectable()
+export class AriaLiveService implements OnDestroy {
+  private _TICK: number = 100;
+  private _ariaLiveElement: HTMLElement;
+  private _document: Document;
+  private _previousTimeout?: number;
+
+  private _id: string = `clr-aria-live-element-${uniqueIdFactory()}`;
```

Comment:
I'm ok with that I also don't like the idea of having the prefix but our code is full of it - So I'm trying to keep it consistence.

```bash
$ ag 'private _' | wc -l
     323
```
If we agree that this will be changed in the feature I'm ok with that :)

Reply:
Agree it should not have prefix. Some of the code is not consistent on the part but we are trying to improve. Remember also for getters and setters the private backing fields typically are prefixed with `_` which is valid.

Reply:
We only try to use underscore prefixes for properties that have getter/setters or public properties that should be only used internally. In all other cases, underscore prefix is not recommended.